superchain/Dockerfile

```diff
@@ -185,7 +185,7 @@ RUN set -ex \
 
 COPY m2-settings.xml $MAVEN_CONFIG/settings.xml
 
-### .NET CLI 2.1 -- https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/dot-net/core-2.1/Dockerfile
+### .NET CLI -- https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/dot-net/core-2.1/Dockerfile
```

Comment:
Well so you removed the 2.1 from before the URL, but what about the one... IN the URL?

Reply:
Actually, there is no codebuild dockerfile for 2.2